src/plugin/debugger/dosdebug.c

```diff
@@ -438,6 +445,13 @@ int main (int argc, char **argv)
   struct timeval timeout;
   const char *rp = getenv("XDG_RUNTIME_DIR");
 
+#ifdef HAVE_LIBREADLINE
+  const char *qp = getenv("READLINE_DISABLE");
```

Comment:
Just use `isatty()` check instead?

Reply:
I suspect that pexpect will defeat that isatty() test as it uses pseudo terminals.

Reply:
Then just a new cmdline option?

Reply:
Btw, if it uses tty then why to bother?

Reply:
Because it's difficult enough to parse for the testing without having to contend with what deadline may or may not be doing to the prompt. Currently it's able to redraw the prompt if it's interrupted by incoming text.

Reply:
Ok.